packages/koa-shopify-auth/src/auth/create-enable-cookies.ts

```diff
@@ -0,0 +1,425 @@
+import {Context} from 'koa';
+
+import Error from './errors';
+import {OAuthStartOptions} from "../types";
+
+export default function createEnableCookies({apiKey}: OAuthStartOptions) {
+  return function enableCookies(ctx: Context) {
+    const HEADING = 'Enable cookies';
```

Comment:
In the [shopify_app PR](https://github.com/Shopify/shopify_app/pull/617) we use the `i18n` gem to provide translations for these. Do we have something similar here? If we don't already have something in this project I'm planning on leaving that for later in the interest of time.

Reply:
We don't have something like that in this project. We have it for react but we don't render any strings in server side node anywhere else so haven't needed it.